tests/test_splitter.py

```diff
@@ -8,8 +8,57 @@
 
 class TestSplitter(TestCase):
 
-    def test_Split(self):
-        sp.Splitter.Split("filepath")
+    def test_Split_one(self):
```

Comment:
Please use lowercase names for functions. Also, what's the difference between "one" and "two"? Can the names for the test functions be more descriptive?